samples/js/flickr/find-photos/src/main.js

```diff
@@ -93,8 +100,20 @@ async function main(): Promise<void> {
         title: v.title,
         tags: new Set(v.tags ? v.tags.split(' ') : []),
         sizes: getSizes(v),
+        datePublished: newStruct('Date', {
```

Comment:
Doesn't feel worth it for two, espec when there's a third right there that's different.